packages/core/src/menu/MenuItem.css

```diff
@@ -1,4 +1,5 @@
 .saltMenuItem {
+  align-items: center;
```

Comment:
This would not be correct as it doesn't follow the current alignment rules of list items, where iconography and other elements should be aligned to the first line of text.